astropy/coordinates/builtin_frames/icrs_cirs_transforms.py

```diff
@@ -8,24 +8,18 @@
 
 from astropy import units as u
 from astropy.coordinates.baseframe import frame_transform_graph
-from astropy.coordinates.transformations import (
-    FunctionTransformWithFiniteDifference,
-    AffineTransform,
-)
 from astropy.coordinates.representation import (
-    SphericalRepresentation,
-    CartesianRepresentation,
-    UnitSphericalRepresentation,
-    CartesianDifferential,
-)
+    CartesianDifferential, CartesianRepresentation, SphericalRepresentation,
+    UnitSphericalRepresentation)
+from astropy.coordinates.transformations import (
+    AffineTransform, FunctionTransformWithFiniteDifference)
```

Comment:
The previous code was `black`-compliant, so if the new code was `black`-compliant too then the diffs here would be smaller.

Reply:
True, but then we'd need to change the isort settings in pyproject.toml and re-sort all the other sorted modules. So I don't think the total diff size will change much.

Reply:
When ``black`` is adopted ``isort`` will have to change and everything re-sorted in one go. The important thing here is that I'm making sure we don't and won't have import cycles.

Reply:
> True, but then we'd need to change the isort settings in pyproject.toml and re-sort all the other sorted modules.

Indeed better to do that when/if `black` is adopted.